test/Main.purs

```diff
@@ -0,0 +1,69 @@
+module Test.Main where
```

Comment:
Thanks for adding this! Can we delete `Examples.purs.hs` now? It seems like this is an equivalent file. You don't have to remove it if you'd prefer not to. But you can if you want.